test/parallel/test-intl-no-icu-data.js

```diff
@@ -1,8 +1,12 @@
 // Flags: --icu-data-dir=test/fixtures/empty/
 'use strict';
-require('../common');
+const common = require('../common');
 const assert = require('assert');
 const config = process.binding('config');
 
+if (!(common.hasIntl && common.hasSmallICU)) {
+  common.skip('not (Intl AND SmallICU)');
+  return;
+}
 assert.deepStrictEqual(Intl.NumberFormat.supportedLocalesOf('en'), []);
```

Comment:
Hmm... I think `Intl.NumberFormat.supportedLocalesOf('en')` with empty icu dir should be `[]` anyway.